### PR TITLE
added documentation for the new absolute_url() and relative_path() Twig functions

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | symfony/symfony#13264 |
| Applies to | 2.7 |
| Fixed tickets | n/a |

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
:+1:

File: reference/twig_reference.rst [hunk 1]
```diff
@@ -355,6 +355,38 @@ Returns the absolute URL (with scheme and host) for the given route. If
...
+.. code-block:: jinja
+
+    {{ relative_path(path) }}
+
+``path``
+    **type**: ``string``
+
+Returns a relative path for the given absolute path (based on the current
+request path). For instance, if the current path is
+``/article/news/welcome.html``, the relative path for path
```
Comment (javiereguiluz) [inline reference/twig_reference.rst]: This line reads a bit strange to me:

```
the relative path for path ``/article/image.png`` is ``../images.png``.
```

What about using this instead:

```
the relative path for ``/article/image.png`` path is ``../images.png``.
```

Comment (WouterJ) [inline reference/twig_reference.rst]: I would remove the second "path" completely: "the relative path for `/article/image.png` is `../images.png`"

Comment (WouterJ) [inline reference/twig_reference.rst]: ping @fabpot

Comment (fabpot) [inline reference/twig_reference.rst]: fixed now

**Comment by WouterJ** [comment]
:+1: again, it's ready!

**Comment by weaverryan** [comment]
Thanks! FYI - I merged this too quickly into master, but cherry-picked back to 2.7 at sha: 8fe906953f031e07159a2a29ce471b3b6afee410

**Pull request merged**